docs/usage.rst

```diff
@@ -176,7 +186,9 @@ Fixtures using a decorator
 
 You can define `Django fixtures`_ using a function decorator. The decorator will
 load the fixtures in the ``before_scenario``, as documented above. It is merely
-a convenient way to keep fixtures close to your steps.
+a convenient way to keep fixtures close to your steps. **Caveat:** if *any* step
+in a scenario uses the fixture decorator, **all** steps in the scenario will
+have the fixture loaded.
```

Comment:
This sounds scary, but isn't this actually somewhat "normal"? What would a "confusing" situation look like?

Hence, do you think that deserves the tone of warning?

Maybe a dedicated "Note" paragraph or box explaining the technical reason, saying, "Just keep in mind ..." (like above) would do the job?

Reply:
In any case, let's stick to semantic markup and text structure:

- The warning is separate unit of information
- It should probably come after the code sample (i.e. first motivate, show, then warn)
- The caveat paragraph should probably be marked up as [admonition](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives)

Reply:
Update pushed.  See if that is better